test/implementation-tests.js

```diff
@@ -140,7 +140,7 @@ describe("Functions with side-effects", () => {
 
     describe("$now() returns timestamp with defined format", function() {
         it("should return result object", function() {
-            var expr = jsonata("$now('[h]:[M][P] [z]')");
+            var expr = jsonata("$now('[h]:[M01][P] [z]')");
```

Comment:
I believe this is unrelated to the rest of the fixes, but agree with the change so not too worried that it's here.